framework/src/task_manager/task_manager_internal.h

```diff
@@ -43,4 +43,39 @@
 #define TM_ALLOC(a)  malloc(a)
 #define TM_FREE(a)   free(a)
 
+struct task_list_s {
+	int pid;
+	void *addr;
+};
+typedef struct task_list_s task_list_t;
+
+struct task_list_data_s {
+	int builtin_idx;
+	int pid;
+	int tm_gid;
```

Comment:
This is same as ```gid``` of ```task_info_list_t```. Let's use same name at same value. I remember Jeongchan said, user can think gid is generic group id, not task manager specific gid. Let's use tm_gid at ```task_info_list_t```.

Reply:
I agree with this. Please check not task_info_list_t, but task_info_t.